### PR TITLE
feat(lint): add eslint & stylelint & babel configuration

### PR DESCRIPTION
**Pull request by maxence-lefebvre**
Added eslint, stylelint, babel configurations.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{
```
Comment (gautier-lefebvre) [inline .babelrc:1]: refactor this file to only include Chrome

File: package.json [hunk 2]
```diff
@@ -101,10 +104,17 @@
     "extends": "react-app"
   },
   "devDependencies": {
+    "babel-preset-env": "^1.6.1",
```
Comment (gautier-lefebvre) [inline package.json]: Install babel@7 instead

File: .babelrc (hunk 1 above)
Comment (maxence-lefebvre) [reply .babelrc:1]: Can you do it ? I am currently debugging electron builds & investigating CI

File: package.json (hunk 2 above)
Comment (maxence-lefebvre) [reply package.json]: Can you do it ? I am currently debugging electron builds & investigating CI

**Review by gautier-lefebvre — APPROVED** [review]

**Pull request merged**